src/app/features/students/components/add-student-modal/add-student-modal.component.html

```diff
@@ -0,0 +1,31 @@
+<nz-modal [(nzVisible)]="isVisible" nzTitle="Add a new student" (nzOnCancel)="handleCancel()" (nzOnOk)="handleAdd()"
```

Comment:
Let's sort tag attributes this way:
1. structure directives (*ngIf/*ngFor)
1. attributes without []
2. attributes with []
3. outputs ()

Like this:
<nz-modal 
nzTitle="Add a new student" 
[nzOkLoading]="isAddLoading" 
[nzOkDisabled]="isAddLoading" 
[nzContent]="modalContent"
[(nzVisible)]="isVisible"  
(nzOnCancel)="handleCancel()" 
(nzOnOk)="handleAdd()">